etna/models/tbats.py

```diff
@@ -0,0 +1,181 @@
+        n_jobs: Optional[int] = None,
+        multiprocessing_start_method: str = "spawn",
+        context: Optional[ContextInterface] = None,
+    ):
+        """Create BATSPerSegmentModel with given parameters.
+        Parameters
+        ----------
+        use_box_cox: bool or None, optional (default=None)
+            If Box-Cox transformation of original series should be applied.
+            When None both cases shall be considered and better is selected by AIC.
+        box_cox_bounds: tuple, shape=(2,), optional (default=(0, 1))
+            Minimal and maximal Box-Cox parameter values.
+        use_trend: bool or None, optional (default=None)
+            Indicates whether to include a trend or not.
+            When None both cases shall be considered and better is selected by AIC.
+        use_damped_trend: bool or None, optional (default=None)
+            Indicates whether to include a damping parameter in the trend or not.
+            Applies only when trend is used.
+            When None both cases shall be considered and better is selected by AIC.
+        seasonal_periods: iterable or array-like of int values, optional (default=None)
+            Length of each of the periods (amount of observations in each period).
+            BATS accepts only int values here.
+            When None or empty array, non-seasonal model shall be fitted.
+        use_arma_errors: bool, optional (default=True)
+            When True BATS will try to improve the model by modelling residuals with ARMA.
+            Best model will be selected by AIC.
+            If False, ARMA residuals modeling will not be considered.
+        show_warnings: bool, optional (default=True)
+            If warnings should be shown or not.
+            Also see Model.warnings variable that contains all model related warnings.
+        n_jobs: int, optional (default=None)
+            How many jobs to run in parallel when fitting BATS model.
+            When not provided BATS shall try to utilize all available cpu cores.
+        multiprocessing_start_method: str, optional (default='spawn')
+            How threads should be started.
+            See https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods
+        context: abstract.ContextInterface, optional (default=None)
+            For advanced users only. Provide this to override default behaviors
+        """
+        self.model = BATS(
+            use_box_cox=use_box_cox,
+            box_cox_bounds=box_cox_bounds,
+            use_trend=use_trend,
+            use_damped_trend=use_damped_trend,
+            seasonal_periods=seasonal_periods,
+            use_arma_errors=use_arma_errors,
+            show_warnings=show_warnings,
+            n_jobs=n_jobs,
+            multiprocessing_start_method=multiprocessing_start_method,
+            context=context,
+        )
+        super().__init__(_TBATSAdapter(self.model))
+
+
+class TBATSPerSegmentModel(_TBATSPerSegmentModel):
```

Comment:
As @alex-hse-repository noticed TBATS can't be used as multi-segment model so we should change the name `TBATSPerSegmentModel` -> `TBATSModel` (for BATS too)